packages/ChangelogLinker/src/ChangelogFormatter.php

```diff
@@ -10,7 +10,7 @@ final class ChangelogFormatter
      * @see https://regex101.com/r/JmKFH1/1
      * @var string
      */
-    private const HEADLINE_PATTERN = '#^(?<headline>[\#]{2,} [\w\d.\-/ ]+)$#m';
+    private const HEADLINE_PATTERN = '#(*ANYCRLF)^(?<headline>[\#]{2,} [\w\d.\-/ ]+)$#m';
```

Comment:
Could you update the regex link above?

Also explain what/why is this added in a comment? So next time it's easier to find

Reply:
will do. For now, I will delete my fork. I messed up something, need to reset

Reply:
No troubles